index.html

```diff
@@ -2,27 +2,40 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <title>Leslie Portfolio project</title>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="stylesheet" href="styles.css" />
   </head>
-  <body class="body">
-    <header class="header">
-      <a class="logo" href="">John Doe</a>
-      <nav class="navigate">
-        <a href=""><img src="images/Union.png" alt="" /></a>
-      </nav>
+  <div class="body">
+    <header>
+      <div class="header mobile-only">
+        <a class="logo" href="">John Doe</a>
```

Comment:
- [ ] [OPTIONAL] I think it will be great if you use your name here so the page can be more customized to you. We want everyone to know whose portfolio this is. 💯️💯️💯️